docs/ossec_alerts.rst

```diff
@@ -527,18 +551,43 @@ securedrop@freedom.press for help.
 
 .. _SecureDrop Support Portal: https://securedrop-support.readthedocs.io/en/latest/
 
-Uncommon OSSEC Alerts
+Daily reports
+^^^^^^^^^^^^^
+
+On days where file integrity checksums have changed or users have logged into ``app``
+or ``mon`` servers, you will receive emails entitled ``Daily report: File changes`` or
+``Daily report: Successful logins``. These emails may be a more digestible format
```

Comment:
Just thinking forward to the time when these docs might be translated, I'd say "convenient" instead of "digestible".